### DISCUSSION TITLE
Updated Next.js (v2) examples

### DISCUSSION THREAD

**Discussion by brentrobbins** [nhost/nhost#101] [Ideas]
I just started on moving my apps to V2 and I feel it could have been easier if there were current V2 Next.js examples I could use for references. I checked out the [Next.js example](https://github.com/nhost/nhost/tree/main/examples/nextjs-apollo) in this repo and it currently doesn't work. Also for some reason when I use the new `nhost` cli for a new project and I use the Next.js frontend template provided by the cli it uses a newer setup, but no features/examples.

**Comment by elitan** [comment]
Good point. Actually all those examples belong to v1 and I think we should remove them for now to avoid confusion and add them back once they've been updated to work with v2.

The templates (e.g. the Next.js template) in our CLI are very bare bone. Do you have any issues with it or you're just missing some more features/examples?

**Reply by brentrobbins** [reply]
@elitan, I do remember running into issues. So I just set up a fresh one to capture the issues:

Right off the bat, I am receiving the following errors:
`client.js?633a:446 Uncaught DOMException: Failed to construct 'WebSocket': The URL 'undefined' is invalid.`
`Warning: Did not expect server HTML to contain a <div> in <div>.`
`react-dom.development.js?61bb:20085 The above error occurred in the <NhostApolloProvider> component:`

So I updated `"@nhost/nhost-js": "^0.1.1"`, to `"@nhost/nhost-js": "^0.3.3",`  no more errors.
